packages/components/src/components/stepper-horizontal/stepper-horizontal-item/stepper-horizontal-item-styles.ts

```diff
@@ -1,69 +1,66 @@
-  )}")`;
-
-const getColors = (
-  state: StepperState,
-  theme: Theme
-): {
+type NumberedCircleColors = {
   primaryColor: string;
-  hoverColor: string;
-  iconColor: string;
   invertedBaseColor: string;
   disabledColor: string;
-} => {
-  const { primaryColor, hoverColor, warningColor, successColor, disabledColor } = getThemedColors(theme);
+};
 
-  const stateToColorMap: Record<StepperState, string> = {
-    current: 'inherit',
-    complete: successColor,
-    warning: warningColor,
-  };
+const getSVGPath = (stepCount: number, numberedCircleColors: NumberedCircleColors, isStateCurrent: boolean): string => {
+  numberedCircleColors = Object.entries(numberedCircleColors).reduce(
+    (result, [key, value]) => ({ ...result, [key]: value.replace(/#/g, '%23') }),
+    {} as NumberedCircleColors
+  );
+
+  const { disabledColor, invertedBaseColor, primaryColor } = numberedCircleColors;
+  const fillColor = isStateCurrent ? invertedBaseColor : disabledColor;
 
-  return {
-    primaryColor,
-    hoverColor,
-    iconColor: stateToColorMap[state],
-    invertedBaseColor: getInvertedThemedColors(theme).primaryColor,
-    disabledColor,
+  const svgCirclePath = `<circle fill="${isStateCurrent ? primaryColor : 'none'}"${
+    isStateCurrent ? '' : ` stroke="${fillColor}"`
+  } stroke-width="1px" cx="12" cy="12" r="9"/>`;
+
+  // Full SVG can be found in ./numbers.svg
```

Comment:
Might be worth commenting where the svg comes from, how it was optimized and how to get the correct paths out of it.